pagopa/pn-frontend
# Pn 3895 fe ricontrollare script dei cookie per tutti i portali

## Short description
Updated all cookies script

## How to test
Start each one of the packages and check that the OT script if correctly uploaded and showed
